test/fixture/throw-anonymous-function.js

```diff
@@ -2,6 +2,6 @@ import test from '../../';
 
 test('throw an uncaught exception', () => {
 	setImmediate(() => {
-		throw () => {};
+		throw function () {};
```

Comment:
Because the arrow function is transformed to an anonymous function by Babel. Changing it back to an arrow function would not change the assertion. I felt this way the assertion was a little easier to understand.